### ISSUE TITLE
[Feature]: Operator can push apps with larger resource requirements by default

### ISSUE THREAD

**Issue by acosta11** [cloudfoundry/korifi#2682]
### Background

**As a** platform operator
**I want** app developers to be able to push apps with larger resource requirements by default
**So that** I do not have to configure the platform to handle apps that push successfully on cf for vms

### Acceptance Criteria

**GIVEN** a default installation of korifi
**WHEN I** push an app with staging resource requirements like the java spring app
**THEN I** the push succeeds

### Dev Notes

We recently added some configuration around app resource limits that caused app pushes to start failing due to low default values. Instead, limits in cf for vms were enforced at the space and org level with quotas, so we would like to instead use something like a kubernetes resource request to implement the appropriate behavior. At the very least, we may just roll back the limit configuration to unblock app pushes.

See related #2663 and #2673. Also see related slack thread:  https://cloudfoundry.slack.com/archives/C0297673ASK/p1689178733777159.
```[tasklist]
### Tasks
- [x] Update kpack buildworkload controller to only set resource requests with current configuration
- [x] Update internal configmap fields names
- [x] Update default values of staging resource requests to either higher numbers or remove them outright
- [x] Revert korifi-ci commit after this fix makes it into main
```

**Comment by georgethebeatle** [comment]
This change resulted in some korifi builds not being able to be scheduled, because  resource requests default to the value of resource limits and it seems that 1G memory request is too big for our clusters. We have therefore reduced the test parallelism a bit to fix CI. Can you please revert [this commit](https://github.com/cloudfoundry/korifi-ci/commit/2db51783592cd4913318cac5704443a818c063b6) once you have a proper fix for this problem. Thanks!

**Comment by acosta11** [comment]
Now that the fix is merged into main, reverting the node configuration and re-flying the pipeline.
